### DISCUSSION TITLE
What is the status of the C++ counter-part implementation?

### DISCUSSION THREAD

**Discussion by zkycaesar** [GSI-CS-CO/chart-fx#664] [General]
I have been using this library in my JavaFx projects for some time, and very impressed by the functions it provides.

Recently, I moved on to Qt, and trying to find a chart lib like this. I noticed that there maybe a planned C++ implementation of this library. I'm really looking forward to this, and wonder what is the status.

**Comment by wirew0rm** [comment]
Hey, thanks for your interest and kind words (and as always, feel free to show of some of your (java) creations in the [show and tell section](https://github.com/fair-acc/chart-fx/discussions/categories/show-and-tell) if you can ;))!

As you noticed correctly, we also moved some of our project to c++ and have been evaluating different UI frameworks for that, during which we also evaluated creating a chart-qt version. This has not come over the early prototype phase since we are now using Dear ImGui for which a very powerful and fast chart implementation already exists with ImPlot. Feel free to use chart-qt as a starting ground for your own experiments, but from our side there are currently no plans to continue into that direction.
